src/pycal/_libcal_math_healpix.cpp

```diff
@@ -17,7 +17,7 @@ void init_math_healpix(py::module & m) {
             py::buffer_info info_vec = vec.request();
             size_t nvec = (size_t)(info_vec.size / 3);
             if ((info_theta.size != info_phi.size) ||
-                (info_theta.size != nvec)) {
+                ( (size_t) info_theta.size != nvec)) {
```

Comment:
The same here. I don't like to force a comparison with a foced casting. We have to compared `info_thera.size` with `nvec` not `info_theta.size` casted. To resolve this warning we have to work on the info_theta.size definitions.

Reply:
Ok, i can solve the problem at the root defining a type casted `info_theta.size`.

Something like:
```
[...]
            py::buffer_info info_theta = theta.request();
            size_t info_theta_size = (size_t) info_theta.size
[...]   
```

Do you think this is a good solution?

Reply:
Uhmm no the problem still remains because it depends on `py::buffer_info` structure. Ok, now I understand better what you said ..... maybe offline we can discuss better